osu.Game.Rulesets.Mania/Skinning/Legacy/ManiaLegacySkinTransformer.cs

```diff
@@ -122,6 +122,9 @@ public override Drawable GetDrawableComponent(ISkinComponentLookup lookup)
                         case ManiaSkinComponents.BarLine:
                             return null; // Not yet implemented.
 
+                        case ManiaSkinComponents.Stage:
+                            return new LegacyStageConfiguration();
```

Comment:
I'm not a huge fan of having to create a separate class for this. It means that a user can't then change the configuration in the lazer editor and have it stick (unless as you mentioned, we were to write to the legacy config... which I'd rather avoid if we can).